docs/noderunner.md

```diff
@@ -0,0 +1,48 @@
+# Node 6 support
+
+Agent tasks can be implemented in PowerShell or Node. The agent currently ships with three versions of Node that tasks can target: 6, 10 & 16.
+
+Since Node 6 has long passed out of the upstream maintenance window, and all officially supported tasks are migrated from Node 6 to Node 10, Node 6 soon will be removed from the agent package. 
+It's also highly recommended to third-party task maintainers migrate tasks to Node 10 or Node 16.
+
+However, to support backward compatibility with the Node 6 tasks we provide self-service methods to install the designated Node runner manually.
+
+## Install Node 6 runner manually
+
+To support the execution of Node 6 tasks agent should be provided with the latest Node 6 version - `6.17.1.0`.
+
+Despite that Node 6 is officially reached the End-of-Life, please, notice that it still can have maintenance updates, so it is required for the agent to get the latest binaries. You can check the currently existing Node versions [here](https://nodejs.org/dist/index.tab).
```

Comment:
This is something not very readable - https://nodejs.org/dist/index.tab

Can we use https://nodejs.org/dist/ here as well?